site/documentation/how-do-i-write-policies/index.md

```diff
@@ -786,6 +786,34 @@ undefined
 
 In some cases, having an undefined result for a document is not desirable. In those cases, policies can use the [Default Keyword](#default) to provide a fallback value.
 
+## <a name="functions"></a> Functions
+
+Rego supports user defined functions. Functions are only accessible within the modules they are defined in. Once defined, a user function behaves like a builtin.
```

Comment:
I think we can generalize this so that functions references behave like rules:

i) within a package, functions can be referred to by name; the compiler should resolve them accordingly
ii) outside a package, functions can be referred to by path; the compiler should resolve them accordingly
iii) functions can be imported like rules (except no data prefix required)